### DISCUSSION TITLE
Start Menu Customization

### DISCUSSION THREAD

**Discussion by FutureNoob** [ramensoftware/windhawk-mods#374] [Mod ideas]
Considering the fact that there's multiple mods that allow for customization of the taskbar, both visually and functionality. Would it be possible for there to be a mod to possibly change the size of the Start Menu? Or remove the recommended section text thingy, or change the layout/pinned icon sizes? I think that would be pretty cool.

(The change size would be especially nice for those who slam the win key to search. 👀)

**Comment by m417z** [comment]
Check out [Windows 11 Start Menu Styler](https://windhawk.net/mods/windows-11-start-menu-styler) 🙂
